src/container/getRelayQueries.js

```diff
@@ -54,7 +54,8 @@ function getRelayQueries(
       var concreteQuery = buildRQL.Query(
         queryBuilder,
         Component,
-        Object.keys(route.params)
+        Object.keys(route.params),
+        Component.getFragment(fragmentName)
```

Comment:
We should also add a test case in `getRelayQueries-test` that ensures route params are passed through properly.